README.md

```diff
@@ -1,15 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# My Portfolio 
 
+> My Portfolio is web showcase of the work i done and the products i have built.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- CSS
+- Github Workflow
 
 ## Live Demo (if available)
```

Comment:
- [OPTIONAL] Remove any subtitles that do not apply to your project so that the README looks as clean as possible